unified-scheduler-pool/src/lib.rs

```diff
@@ -709,6 +709,33 @@ fn disconnected<T>() -> Receiver<T> {
     crossbeam_channel::unbounded().1
 }
 
+#[cfg_attr(doc, aquamarine::aquamarine)]
+/// The concrete scheduler instance along with 1 scheduler and N handler threads
+///
+/// ### Life cycle and ownership movement across crates of a particular scheduler
+///
+/// ```mermaid
```

Comment:
currently, rendered like this:

![image](https://github.com/user-attachments/assets/21a5a967-6328-49eb-94f7-25c8485a23b2)

needs accompanying wall of text to explain subtleties. :)